### PR TITLE
Feature/sem 050

### PR DESCRIPTION
**Pull request by ColqueApazaGualberto**
Como usuario quiero ver un label que redireccione a las secciones correspondientes.
Test 1 for SEM-050 Atom Review

### CODE REVIEW

**Comment by DanX3N0** [comment]
PR:
image

It should be just one component with its respective variations

File: src/ui/components/atoms/label-variables/Variables.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  const getColorClassName = () => {
+    switch (colorVariant) {
+      case 'main':
+        return 'variables--main';
+      case 'secondary':
+        return 'variables--secondary';
+      default:
+        return 'variables--primary';
+    }
+  };
```
Comment (Ditmar) [inline src/ui/components/atoms/label-variables/Variables.tsx:14]: I don't see it necessary to create a function to translate the data there are too many different approaches to this more easily.
something like this.
```
const colorClassNames: Record<string, string> = {
  main: 'variables--main',
  secondary: 'variables--secondary',
  default: 'variables--primary'
};
```
and to use 
const className = colorClassNames[colorVariant]

File: src/ui/components/atoms/label-variables/styles.scss [hunk 2]
```diff
@@ -0,0 +1,83 @@
+@import '../../../../globals/variables';
+@import '../../../../globals/mixins';
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap');
```
Comment (Ditmar) [inline src/ui/components/atoms/label-variables/styles.scss:3]: please replace this code by the embedded fonts

**Review by Ditmar — CHANGES_REQUESTED** [review]
read the comments

**Review by Ditmar — APPROVED** [review]

**Pull request merged**